editor/components/post-featured-image/style.scss

```diff
@@ -1,33 +1,38 @@
 .editor-post-featured-image {
-	padding: 10px 0 0;
+	padding: 0;
 
 	.spinner {
 		margin: 0;
 	}
-}
-
-.editor-post-featured-image__toggle {
-	text-decoration: underline;
-	color: theme( secondary );
 
-	&:focus {
-		box-shadow: none;
-		outline: none;
+	// Space consecutive buttons evenly.
+	.components-button + .components-button {
+		margin-top: 1em;
```

Comment:
Eventually I'll propose sorting our CSS attributes alphabetically; until then I'll live with this 😉

Reply:
:) I like your style.

Though elaborate a bit — are you referring to the top margin going before the right margin? For north south east west things I always go clockwise, i.e. top, right, bottom, left.